### ISSUE TITLE
Fixing the nan Issue in the VAE

### ISSUE THREAD

**Issue by lucasfijen** [JMitnik/FacialDebiasing#10]
So there are a couple of things we could try to prevent nans from happening on the long run:

There are some useful infomation about why nan problem could happen:
1.the learning rate
2.sqrt(0)
3.ReLU->LeakyReLU

or gradient clipping
